release-notes/major11/11.0.0/valtimo-backend-libraries.md

```diff
@@ -43,7 +43,13 @@ The following breaking changes were introduced:
   * `/api/v1/public/task/{taskDefinitionId}/resource/pre-signed-url/{fileName}`
   * `/api/v1/public/task/{taskDefinitionId}/resource/{resourceId}`
 
-* **Breaking change2**
+* **Custom plugin scanning**
+
+  Scanning for plugins has changed. Only whitelisted packages will be scanned for possible custom plugins. Custom
+  plugins might not be found anymore when the plugin's package name is not whitelisted. To make sure all custom plugins
```

Comment:
Maybe add sentence after "when the plugin's package name is not whitelisted"

By default only Valtimo default plugins, and plugins in the classpath of your spring application main class will be found.